apps/console/src/features/claims/api/claims.ts

```diff
@@ -530,6 +538,7 @@ export const getServerSupportedClaimsForSchema = (id: string): Promise<ServerSup
  * Fetch all local claims.
  *
  * @param params - limit, offset, sort, attributes, filter.
+ 
```

Comment:
Let's add the missing * here.

Reply:
Added in https://github.com/wso2/identity-apps/pull/6597.